src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { useRef, useEffect } from "react";
+
+const Input = ({ onEditComplete, defaltValue }) => {
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.value = defaltValue;
+    inputRef.current.focus();
+    inputRef.current.onblur = () => {
+      onEditComplete(inputRef.current.value);
+    };
+    (inputRef.current.onkeypress = (e) => {
+      if (e.key === "Enter") {
+        onEditComplete(inputRef.current.value);
+      }
+    }),
+      [];
+  });
```

Comment:
```suggestion
    inputRef.current.onkeypress = (e) => {
      if (e.key === "Enter") {
        onEditComplete(inputRef.current.value);
      }
    }
  },[]);
```
おそらく括弧の対応が崩れて事故っております

Reply:
@MoAI522 

対応しました

b7f71ddef14f8b86cbe02d44b521e6b4352d706c